Dockerfile

```diff
@@ -0,0 +1,140 @@
+  libssl-dev=1.1.1n-0+deb11u4 \
+  pkg-config=0.29.2-1 \
+  unzip=6.0-26+deb11u1 \
+  wget=1.21-1+deb11u1 \
+  software-properties-common=0.96.20.2-2.1 
+RUN add-apt-repository -y ppa:openjdk-r/ppa && \
+  apt-get install -y --no-install-recommends \
+  openjdk-17-jdk=17.0.6+10-1~deb11u1
+
+#install Gradle
+RUN wget -q https://services.gradle.org/distributions/gradle-7.2-bin.zip \
+  && unzip gradle-7.2-bin.zip -d /opt \
+  && rm gradle-7.2-bin.zip
+
+# Set Gradle in the environment variables
+ENV GRADLE_HOME=/opt/gradle-7.2
+ENV PATH=/opt/gradle-7.2/bin:$PATH
+ENV JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8
+
+COPY . /radixdlt
+WORKDIR /radixdlt
+USER root
+RUN SKIP_NATIVE_RUST_BUILD=TRUE gradle clean build -x test -Pci=true -PrustBinaryBuildType=release
+
+USER nobody
+
+# Exporting babylon-node binary
+FROM scratch AS binary-container
+COPY --from=binary-build-stage /radixdlt/core/build/distributions /
+
+# Building core-rust library
+FROM lukemathwalker/cargo-chef:0.1.52-rust-1.68.0 AS chef
+LABEL org.opencontainers.image.authors="devops@radixdlt.com"
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    clang=1:11.0-51+nmu5 \
+    libmpc-dev=1.2.0-1 \
+    libmpfr-dev=4.1.0-3 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+WORKDIR /app
+WORKDIR /app
+
+FROM chef AS planner
+COPY core-rust ./
+RUN cargo chef prepare --recipe-path recipe.json
+
+FROM chef AS library-builder 
+ENV RUST_BACKTRACE=1
+RUN apt-get update && apt-get install libclang-dev=1:11.0-51+nmu5 -y --no-install-recommends
+COPY --from=planner /app/recipe.json recipe.json
+
+ARG TARGETPLATFORM
+ARG RUST_PROFILE=release
+COPY docker/scripts/cargo_build_by_platform.sh /opt/radixdlt/cargo_build_by_platform.sh
+RUN /opt/radixdlt/cargo_build_by_platform.sh chef $TARGETPLATFORM release
+# Build application
+COPY core-rust ./
+RUN /opt/radixdlt/cargo_build_by_platform.sh build $TARGETPLATFORM release
+
+# Exporting core-rust library
+FROM scratch as library-container
+COPY --from=library-builder /libcorerust.so /
+
+# Building the application container
+FROM debian:11-slim as app-container
+LABEL org.opencontainers.image.authors="devops@radixdlt.com"
+
+RUN apt-get update -y && \
+    apt-get -y --no-install-recommends install \
+    openjdk-17-jre-headless=17.0.6+10-1~deb11u1 \
+    unzip=6.0-26+deb11u1 \
+    daemontools=1:0.76-7 \
+    libssl-dev=1.1.1n-0+deb11u4 \
+    software-properties-common=0.96.20.2-2.1 \
+    gettext-base=0.21-4 && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY docker/scripts/fix-vulnerabilities.sh /tmp
+RUN /tmp/fix-vulnerabilities.sh
+
+# create configuration automatically when starting
+COPY docker/scripts/config_radixdlt.sh /opt/radixdlt/config_radixdlt.sh
+
+# copy configuration templates
+WORKDIR /home/radixdlt
+COPY docker/config/ /home/radixdlt/
+
+# Add script to install optional network utils
+COPY docker/scripts/install_network_utils.sh /opt/radixdlt/install_network_utils.sh
```

Comment:
Do you think it might be worth setting up a [motd](https://en.wikipedia.org/wiki/Message_of_the_day) to alert an ssh'ing user that they may wish to run this script?

Reply:
@dhedey What do you think of the message I just added?

Reply:
Really nice, thanks! ✨ 

Couple of minor thoughts:
* There are some Github Actions lint notices that we might need to work around - possibly extracting a set_motd.sh script (which can use a heredoc) or `motd.txt` might make it easier to edit 👍 
* "However we left you the instructions to install them in this file:" doesn't like they're written instructions rather than bash comments, I'd suggest instead "Install them manually by running /opt/radixdlt/install_network_utils.sh"